tests/test_cli.py

```diff
@@ -226,6 +226,22 @@ def test_matched(self, script_runner: ScriptRunner) -> None:
         assert result.success
         assert result.stdout[0].isdigit(), "Output does not start with a number"
 
+    @pytest.mark.vcr
+    def test_fields(self, script_runner: ScriptRunner) -> None:
+        args = [
+            "stac-client",
+            "search",
+            STAC_URLS["EARTH-SEARCH"],
+            "-c",
+            "cop-dem-glo-30",
+            "--max-items",
+            "1",
+            "--fields=-geometry,-assets",
+        ]
+        result = script_runner.run(args, print_result=True)
+        assert result.success
+        assert "assets" not in result.stdout
```

Comment:
Should probably check to make sure `geometry` isn't there as well, just for completeness?

Reply:
indeed